### PR TITLE
sig-ddl: update the requirement of committers

### PR DESCRIPTION
**Pull request by tangenta**

### CODE REVIEW

File: special-interest-groups/sig-ddl/roles-and-organization-management.md [hunk 1]
```diff
@@ -28,10 +28,10 @@ Requirements:
 
 Requirements:
 
-* Have at least 40 PRs (at least 10 PRs in tidb repo and at least 8 PRs in parser repo) merged.
-* Have fixed at least 5 issues whose difficulty is medium or above.
+* Have at least 20 PRs (at least 10 PRs in tidb repo and at least 8 PRs in parser repo) merged.
```
Comment (djshow832) [inline special-interest-groups/sig-ddl/roles-and-organization-management.md:31]: 10 PRs in tidb and 8 PRs in parser? Then it's very limited.

**Review by wjhuang2016 — COMMENTED** [review]
LGTM

File: special-interest-groups/sig-ddl/roles-and-organization-management.md (hunk 1 above)
Comment (tangenta) [reply special-interest-groups/sig-ddl/roles-and-organization-management.md]: Updated.

**Review by djshow832 — COMMENTED** [review]
LGTM

**Review by djshow832 — APPROVED** [review]

**Pull request merged**